<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>731</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>114</width>
          <height>369</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QListWidget操作</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>161</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QToolButton" name="btnInitList_1">
            <property name="text">
             <string>初始化列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnClearList_1">
            <property name="text">
             <string>清除列表</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnInsertPage_1">
            <property name="text">
             <string>插入页</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnAdd_1">
            <property name="text">
             <string>添加项</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnDeleteCurrentPage_1">
            <property name="text">
             <string>删除当前页</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>114</width>
          <height>369</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTreeWidget操作</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>114</width>
          <height>369</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTableWidget操作</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>QListWidget</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>60</y>
           <width>431</width>
           <height>51</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>381</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QToolButton" name="btnSelectItem">
             <property name="text">
              <string>项选择</string>
             </property>
             <property name="icon">
              <iconset resource="Resource.qrc">
               <normaloff>:/image/select.ico</normaloff>:/image/select.ico</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QToolButton" name="btnPartialSelect">
               <property name="text">
                <string>反选  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnSelectNotAtALL_1">
               <property name="text">
                <string>全不选</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnSelectAll_1">
               <property name="text">
                <string>全选  </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>0</y>
           <width>431</width>
           <height>41</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>当前项变化</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="varEditText">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>10</y>
            <width>181</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>130</y>
           <width>431</width>
           <height>251</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>item1</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="background">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>255</red>
             <green>85</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
          <property name="foreground">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>255</red>
             <green>85</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Iterm2</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>QTreeWidget</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>QTableWidget</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actInitList"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actClearList"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actInsErt"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actAdd"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actClearCurrent"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <action name="actInitList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="初始化列表" resource="Resource.qrc">
     <normaloff>:/image/initlist.ico</normaloff>:/image/initlist.ico</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
  </action>
  <action name="actClearList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="清除列表" resource="Resource.qrc">
     <normaloff>:/image/clear.ico</normaloff>:/image/clear.ico</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actInsErt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="插入项" resource="Resource.qrc">
     <normaloff>:/image/insert.ico</normaloff>:/image/insert.ico</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
  </action>
  <action name="actAdd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="添加项" resource="Resource.qrc">
     <normaloff>:/image/add.ico</normaloff>:/image/add.ico</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
  </action>
  <action name="actClearCurrent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="清除当前项" resource="Resource.qrc">
     <normaloff>:/image/delete.ico</normaloff>:/image/delete.ico</iconset>
   </property>
   <property name="text">
    <string>清除当前项</string>
   </property>
   <property name="toolTip">
    <string>清除当前项</string>
   </property>
  </action>
  <action name="actSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="项选择">
     <normaloff>select.ico</normaloff>select.ico</iconset>
   </property>
   <property name="text">
    <string>项选择</string>
   </property>
   <property name="toolTip">
    <string>项选择</string>
   </property>
  </action>
  <action name="actExit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="退出" resource="Resource.qrc">
     <normaloff>:/image/exit.ico</normaloff>:/image/exit.ico</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actSelectAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="actSelectNAll">
   <property name="text">
    <string>全不选</string>
   </property>
   <property name="toolTip">
    <string>全不选</string>
   </property>
  </action>
  <action name="actSelectInvs">
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>反选</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actClearList</sender>
   <signal>triggered()</signal>
   <receiver>listWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
